organisms/organisation-unit-tree/new-features/select-relative-units.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Select units relative to the user's org unit, 'relative units'
+
+  Background:
+    Given the org unit tree has relative selection enabled
+
+  Scenario: Relative selection(s) is made
+    Given the user has opened the relative selection menu
+    When the user selects one or more relative units
+    Then a visual indication of the selection type is presented
+    And the org unit tree is disabled
+
+  Scenario: Non-relative selections persist after a relative selection is made and cleared
```

Comment:
I just read this scenario in detail after writing the comments above, I'm a bit confused now:
`persists` to me means the also the selection is kept (also visibly), but the last sentence of this scenario indicates that org units selected previous to the relative selection are not selected while relative selection is enabled. Maybe the wording is a bit odd here, could be:

`Scenario: Non-relative selections previous to relative selection are restored when relative selection is cleared`
(bit too long, but you get what I mean)

Reply:
Yes, the wording was odd. I've adapted yours instead, I don't think it was too long at all.